tests/workflows/basic_subworkflow_deployment/tests/test_workflow.py

```diff
@@ -109,6 +109,25 @@ def generate_subworkflow_events(*args: Any, **kwargs: Any) -> Iterator[WorkflowS
     assert parent_context["node_definition"]["name"] == expected_context["name"]
 
 
+def test_subworkflow_deployment_node_with_base_inputs(vellum_client):
+    from tests.workflows.basic_subworkflow_deployment.workflow_with_base_inputs import (
+        ExampleSubworkflowDeploymentNodeWithBaseInputs,
+    )
+
+    node = ExampleSubworkflowDeploymentNodeWithBaseInputs()
+    compiled_inputs = node._compile_subworkflow_inputs()
```

Comment:
Instead of calling private methods, which you should NEVER do in tests, see if we could just pytest parameterize the `test_run_workflow__happy_path` by the workflow class we want to test (`WorkflowWithBaseInputsSubworkflow` vs the original one)